index.html

```diff
@@ -136,10 +136,15 @@ <h2>Bronze Sponsors</h2>
   
         <ul class="sponsor__list">
           <li>
-            <a href="https://https://9elements.com" target="_blank" class="sponsor__logo">
+            <a href="https://9elements.com" target="_blank" class="sponsor__logo">
               <img src="./assets/sponsors/9elements_logo_black.svg" alt="9elments company logo">
             </a>
           </li>
+          <li>
+            <a href="https://flexvelop.com" target="_blank" class="sponsor__logo">
+              <img src="./assets/sponsors/flexvelop.svg" alt="flexvelop company logo" style="min-height: 250px;">
```

Comment:
The inline style is needed because the image has no width and height info.

In the `index.html` I choose the `min-height` info so that images are connected near the bottom. See screenshot